init.c

```diff
@@ -466,6 +466,21 @@ int main(int argc, char *argv[])
 
 	// create pv thread
 	pv_pid = fork();
+	if (pv_pid > 0 && getuid() == 0) {
```

Comment:
aren't we doing the same in this if and else bodies?

Reply:
no the other else is for the child process (e.g. pv_pid == 0) not > 0

Reply:
well, I mean what's inside of the body of the if and the else is the same
```
	if (pv_system_oom_adjust(
			    pv_config_get_vm_oom_qos_guaranteed())) {
			printf("ERROR: oom adjust failed: '%s'\n",
			       strerror(errno));
			return -1;
```